cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from cinema.views import movie_list, movie_detail
+
+
+urlpatterns = [
+    path("movies", movie_list, name="movie_list"),

```

Comment:
see the checklist:
- endpoints should end with the slash
- hyphen should be the delimiter in the url names

Reply:
Thank you)